examples/client/client.c

```diff
@@ -42,18 +42,13 @@
 
 #include <wolfssl/wolfcrypt/settings.h>
 
-#if !defined(WOLFSSL_TRACK_MEMORY) && !defined(NO_MAIN_DRIVER)
-    /* in case memory tracker wants stats */
-    #define WOLFSSL_TRACK_MEMORY
-#endif
-
 #include <wolfssl/ssl.h>
 
 #include <wolfssl/test.h>
 
 #include "examples/client/client.h"
 
-#ifdef WOLFSSL_ASYNC_CRYPT
+#ifndef HAVE_FIPS
```

Comment:
It has to do with the "wc_InitRng_ex" call which requires a devId parameter. The FIPS RNG doesn't support the wc_InitRng_ex function. I did clean this up for clarity so the devId is only there for WOLFSSL_ASYNC_CRYPT now and the FIPS logic is at the wc_InitRng_ex call (line 278).